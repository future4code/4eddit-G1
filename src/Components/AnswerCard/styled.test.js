import React from "react";
import renderer from "react-test-renderer";
import { AnswerCardMainContainer, HeaderPostContent, UserName, BodyPostContent, FooterPostContent, ButtonArea, ComentWords, UpArrow, DownArrow } from './styled'

describe("Styleds", () => {
    // Mesma coisa do que test()
    it("AnswerCardMainContainer", () => {
      const tree = renderer.create(<AnswerCardMainContainer />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("DownArrow", () => {
      const tree = renderer.create(<DownArrow />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("ButtonArea", () => {
      const tree = renderer.create(<ButtonArea />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("ComentWords", () => {
      const tree = renderer.create(<ComentWords />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("UpArrow", () => {
      const tree = renderer.create(<UpArrow />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("HeaderPostContent", () => {
      const tree = renderer.create(<HeaderPostContent />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("UserName", () => {
      const tree = renderer.create(<UserName />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("BodyPostContent", () => {
      const tree = renderer.create(<BodyPostContent />).toJSON();
      expect(tree).toMatchSnapshot();
    });
    // Mesma coisa do que test()
    it("FooterPostContent", () => {
      const tree = renderer.create(<FooterPostContent />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });