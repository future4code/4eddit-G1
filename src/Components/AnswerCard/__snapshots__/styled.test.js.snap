// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendButton should match snapshot 1`] = `
<div
  className="sc-bdVaJa bhJRfQ"
/>
`;

exports[`SendButton should match snapshot 2`] = `
<div
  className="sc-htpNat htxEGc"
/>
`;

exports[`SendButton should match snapshot 3`] = `
<span
  className="sc-bZQynM iZGwtf"
/>
`;

exports[`SendButton should match snapshot 4`] = `
<img
  className="sc-gzVnrw dCULKn"
/>
`;

exports[`SendButton should match snapshot 5`] = `
<div
  className="sc-bwzfXH fICcRd"
/>
`;

exports[`SendButton should match snapshot 6`] = `
<span
  className="sc-EHOje jKoLEy"
/>
`;

exports[`SendButton should match snapshot 7`] = `
<div
  className="sc-bxivhb hACKXe"
/>
`;

exports[`SendButton should match snapshot 8`] = `
<div
  className="sc-ifAKCX cxwvfZ"
/>
`;

exports[`SendButton should match snapshot 9`] = `
<div
  className="sc-ifAKCX cxwvfZ"
/>
`;

exports[`Styleds AnswerCardMainContainer 1`] = `
<div
  className="sc-bdVaJa bhJRfQ"
/>
`;

exports[`Styleds BodyPostContent 1`] = `
<div
  className="sc-bxivhb hACKXe"
/>
`;

exports[`Styleds ButtonArea 1`] = `
<div
  className="sc-htpNat htxEGc"
/>
`;

exports[`Styleds ComentWords 1`] = `
<span
  className="sc-bZQynM iZGwtf"
/>
`;

exports[`Styleds DownArrow 1`] = `
<img
  className="sc-htoDjs etQmDw"
/>
`;

exports[`Styleds FooterPostContent 1`] = `
<div
  className="sc-ifAKCX cxwvfZ"
/>
`;

exports[`Styleds HeaderPostContent 1`] = `
<div
  className="sc-bwzfXH fICcRd"
/>
`;

exports[`Styleds UpArrow 1`] = `
<img
  className="sc-gzVnrw dCULKn"
/>
`;

exports[`Styleds UserName 1`] = `
<span
  className="sc-EHOje jKoLEy"
/>
`;
